src/pages/index.tsx

```diff
@@ -6,16 +6,18 @@ import { graphql } from 'gatsby'
 import Img from 'gatsby-image'
 
 import { Layout } from '../components/templates/Layout'
-import SEO from '../components/base/seo/seo'
+// import SEO from '../components/base/seo/seo'
 
-import Card from '../components/molecules/card/card'
-import Features from '../components/organisms/features/features'
-import Signup from '../components/organisms/signup/signup'
-import { CtaGrid } from '../components/organisms/ctaGrid/CtaGrid'
+// import Card from '../components/molecules/card/card'
+// import Features from '../components/organisms/features/features'
+// import Signup from '../components/organisms/signup/signup'
```

Comment:
Are these TODOs or should they be removed?

Reply:
Yes, I'm leaving the old content for now. I'll remove all commented code once the homepage is completed with the new content/designs